components/brave_rewards/resources/page/promos/index.tsx

```diff
@@ -20,11 +18,42 @@ export interface Promo {
   supportedLocales: string[]
 }
 
-export const activePromos = [
-  'tap-network'
-]
+export const getActivePromos = (rewardsData: Rewards.State) => {
+  let promos = ['tap-network']
```

Comment:
Curious if you prefer `let` or `const`, from a style perspective. I don't really mind either way.

Reply:
let for variables that you can't change and then const for constants